app/auth/token-validator.js

```diff
@@ -74,9 +74,11 @@ const validateEnforcement = async (request, username, payload) => {
   const validToken = await checkTokenOnline(username, token)
 
   if (validToken) {
-    const enabled = await isAccountEnabled(username)
+    const [enabled, userAccount] = await isAccountEnabled(username)
```

Comment:
I'm not proposing a change here but just wanted to know your thought process ... I would have defaulted to { enabled, userAccount } i.e. returning a compound object, not an array. Is there any benefit in returning an array?